saleor/core/tracing.py

```diff
@@ -5,10 +5,22 @@
 from graphql import ResolveInfo
 
 DEFAULT_RESOLVERS = {default_resolver, GlobalID.id_resolver}
+NO_TRACE_ATTR = "__no_trace"
+
+
+def no_trace(func):
+    def wrapper(*args, **kwargs):
+        info = next(arg for arg in args if isinstance(arg, ResolveInfo))
+        setattr(info.context, NO_TRACE_ATTR, True)
```

Comment:
investigated and this propagates to other fields after first decorated field is resolved, will have to be changed.